src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -95,6 +95,15 @@ class HaAutomationPicker extends LitElement {
   private _columns = memoizeOne(
     (narrow: boolean, _locale): DataTableColumnContainer => {
       const columns: DataTableColumnContainer = {
+        icon: {
+          title: "",
+          label: this.hass.localize(
+            "ui.panel.config.automation.picker.headers.state"
```

Comment:
State as header for an icon?

Reply:
🤷 I agree it is odd, yet, we do the same in scenes & scripts too. Hence I kept it consistent.

![image](https://user-images.githubusercontent.com/195327/197746038-ce4bab30-5e53-4377-8788-2c90ea52efcd.png)